app/concepts/movies/use_cases/show_index.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Movies
+  module UseCases
+    class ShowIndex
```

Comment:
I think you don't need that use case, you can simply use `MovieRepository.new.find_movie(id)` instead.